libs/platform/src/lib/components/approval-flow/approval-flow.component.scss

```diff
@@ -18,6 +18,13 @@ $block: approval-flow;
         cursor: pointer;
     }
 
+    fd-avatar:focus {
+        outline-offset: 0.0625rem;
+        outline-width: 0.0625rem;
+        outline-style: dotted;
+        outline-color: #0854A0;
```

Comment:
please add `outline-color: var(--sapContent_FocusColor, #0854A0); ` here and in other places in this file

Reply:
fixed